rules/gb-model.smk

```diff
@@ -10,6 +10,11 @@ from pathlib import Path
 import numpy as np
 
 
+wildcard_constraints:
+    flexibility_type="fes_ev_dsm|fes_ev_v2g|fes_residential_dsr|fes_services_dsr",
+    data_type="storage|unmanaged_charging",
```

Comment:
Use `ev_data_type` since otherwise it risks clashing with other uses of such a generic wildcard name in future